shared/cluster.go

```diff
@@ -184,14 +188,14 @@ func kubectlCmdOnNode(cmd string) (string, error) {
 }
 
 // FetchClusterIP returns the cluster IP and port of the service.
-func FetchClusterIP(namespace string, serviceName string) (string, string, error) {
-	ip, err := RunCommandHost("kubectl get svc " + serviceName + " -n " + namespace +
+func FetchClusterIP(namespace string, serviceName string) (ip, port string, err error) {
```

Comment:
Should be consistent here and define the type for `ip` also:
`func FetchClusterIP(namespace string, serviceName string) (ip string, port string, err error) {`

Reply:
This is a common standard and good practice in golang. If you 2 params is the same type you declare it once

Reply:
Ah good to know! Learn something new every day 😄